exercise/templates/exercise/exercise.html

```diff
@@ -66,18 +66,14 @@ <h3 class="text-center">{% trans "Submission received." %}</h3>
 <script src="{% static 'exercise/poll.js' %}"></script>
 <script src="{% static 'exercise/group.js' %}"></script>
 <script src="{% static 'exercise/chapter.js' %}"></script>
+<script src="{% static 'exercise/language_link.js' %}"></script>
 <script>
-// Visualize loading in case of slow POST operations.
-/*$(function() {
-    var forms = $("#exercise-page-content").find("form:not([data-aplus-exercise] form)");
-    forms.on("submit", function(event) {
-        $("#submit-progress").removeClass("hide");
-    });
-});*/
-
 $(function() {
     // LTI exercises
     $('[data-external-launch]').aplusExternalLauncher();
+
+    // Links that change language
+    set_language_links('{% url 'set_language' %}');
```

Comment:
I just removed this block in local change to get rid of inline javascript. We probably should use link or similar element or just hardcode the url in the js file ( :( ).

Reply:
Probably better to hardcode the url in that case. There doesn't seem to be any suitable `rel` value for links or any other appropriate metadata tags.

---

On another note, life would be much easier if there was a sane way to pass django template variables to javascript